frontend/components/dashboard/new-session.tsx

```diff
@@ -83,10 +153,40 @@ export const NewSession = () => {
                     </SelectContent>
                 </Select>
             </div>
-
             <Button className="mt-4 bg-purple-600 hover:bg-[#A78BFA]" onClick={handleMatchmaking}>
                 Start matchmaking
             </Button>
+
+            {modalData.isOpen && (
+                <CustomModal className={`h-3/5 w-2/5 rounded-lg`}>
```

Comment:
height shouldn't be fixed here, testing on 14inch laptop causes children to overflow as the height is too small